pkg/nodeconfig/nodeconfig.go

```diff
@@ -77,8 +77,8 @@ type ErrWriter struct {
 }
 
 func (ow OutWriter) Write(p []byte) (n int, err error) {
-	// log debug
-	ow.log.V(1).Info(string(p))
+	// log info
+	ow.log.Info(string(p))
```

Comment:
Sample output below. There is only a minor cosmetic issue, it adds a [line break](https://github.com/openshift/windows-machine-config-operator/blob/master/vendor/k8s.io/kubectl/pkg/drain/drain.go#L292).

```
	INFO	controllers.configmap	processing	{"instances in": "windows-instances"}
	DEBUG	wc 172.31.251.202	initializing SSH connection
	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-scbm8

	INFO	nc 172.31.251.202	evicting pod default/win-webserver-wm-6756f7468f-zmnc4


```